platform/linux-generic/odp_schedule_basic.c

```diff
@@ -1196,7 +1196,7 @@ static inline int balance_spread(int grp, int prio, int cur_spr)
 	return new_spr;
 }
 
-static inline int copy_from_stash(odp_event_t out_ev[], uint32_t max)
+static inline int copy_from_stash(odp_event_t *restrict out_ev, uint32_t max)
```

Comment:
Could also use ```static inline int copy_from_stash(odp_event_t out_ev[restrict], uint32_t max)``` to maintain the same style but either method is OK for me.

Reply:
Yes, but I felt that it is a bit weird syntax and harder to understand. But maybe it is just me. Did not change it in v2 though.